.github/workflows/build.yml

```diff
@@ -56,7 +56,9 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
-      run: python -m pip install tox
+      run: | 
+       python -m pip install --upgrade pip
```

Comment:
Our github actions workflows were actually broken, so the fix was to first upgrade pip then install tox.